eclipse-theia/theia
# monaco: fix `activeItem` handling

#### What it does


Fixes https://github.com/eclipse-theia/theia/issues/10454.

The pull-request fixes an issue where the `activeItem` was not being respected when calling the `showQuickPick` API.
The `activeItem` is now respected and the item will be defaulted in the list and highlighted. The issue was that we were setting the `activeItem` before the `items` list had been set which resulted in nothing being set.

#### How to test


1. confirm that `Preferences: File Icon Theme` will default to and highlight the active file icon theme when the menu is opened
2. confirm that `Preferences: Color Theme` will default to and highlight the active color theme when the menu is opened
3. confirm for a given extension that `Install Another Version...` will default to and highlight the `current` version
4. confirm that `Configure Display Language` highlights the current language (regression test)

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-off-by: vince-fugnitto <vincent.fugnitto@ericsson.com>
